jellyfin-api/src/commonMain/kotlin-generated/org/jellyfin/sdk/api/operations/AudioApi.kt

```diff
@@ -150,6 +152,11 @@ public class AudioApi(
 		val pathParameters = buildMap<String, Any?>(1) {
 			put("itemId", itemId)
 		}
+		require(container == null || Regex("""^[a-zA-Z0-9\-\._,|]{0,40}$""").matches(container)) { """Parameter "container" must match ^[a-zA-Z0-9\-\._,|]{0,40}$.""" }
```

Comment:
I know it's because of the generator, but I'm really not a fan of this repeated Regex object creation.
Maybe you could bundle an object class in the SDK which a) validates a certain string against a given string pattern and b) keeps a map of patterns and their respective Regex instances to use. You could also inline the `require` into that new helper function.

Reply:
My idea was to just use the pattern as a key. That should work and result in very few RegEx instances.

Reply:
So something like this?

```kotlin
object RegexPatterns {
    val `^[a-zA-Z0-9\-\._,|]{0,40}$` = Regex("""^[a-zA-Z0-9\-\._,|]{0,40}$""")
}
```

Reply:
No, sorry, I wasn't obvious in my explanation. A code example should do:
```kt
object RegexValidator {
    private val regexPatterns = HashMap<String, Regex>()

    fun validate(pattern: String, input: String): Boolean {
        val regex = regexPatterns.getOrPut(pattern) { Regex(pattern) }
        return regex.matches(input)
    }
}
```

Reply:
Fine for now, but raised an issue to improve this later: #864.